README.md

```diff
@@ -352,6 +352,15 @@ trakt_tv:
         max_medias: 3
 ```
 
+#### sort_by Options
+
+For any sensor that allows sorting, the following are the options available
+
+`rating`, `rating_trakt`, `rank`, `released`, `runtime`, `listed_at`, `last_activity_date`, `trakt`
```

Comment:
Could you please keep each pull request focused on a single goal?
From the title, this one seems to be about adding the last activity date, but it also includes some unrelated changes (like refactoring the README and adding a test about Trakt sorting in conftest).

To keep the review process simple and clear, could you please remove those extra changes and keep only the parts related to the last activity feature?

If you’d like, feel free to open separate PRs for the other improvements !

Reply:
Indeed, it is not clear sorry, the "things in common" was the same code copied and pasted in two different PR. The chunks were not small, it was the same code with a slight change.

Reply:
`I just supposed to copy/paste the same documentation in multiple places?`, yes and if you want to factorize maybe in another PR and we can discuss later.